funnel/models/label.py

```diff
@@ -85,10 +85,26 @@ class Label(BaseScopedNameMixin, db.Model):
     def restricted(self):
         return self.parent_label._restricted if self.parent_label else self._restricted
 
+    @restricted.setter
+    def restricted(self, value):
+        self._restricted = value
```

Comment:
This should refuse to be set if it has a parent, as the parent's value overrides.